<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GOColorModeDesigner</class>
 <widget class="QWidget" name="GOColorModeDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atom Label</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="generalGB">
         <property name="title">
          <string>General</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Specific</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Mode:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="modesCB"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="userColorTab">
      <property name="styleSheet">
       <string notr="true">QPushButton {
	border: 1px solid #101010;
	border-radius: 5px;
    padding: 2px;
    margin: 2px;
}</string>
      </property>
      <attribute name="title">
       <string>User</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <item>
          <spacer name="horizontalSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QColorPushButton" name="TBEleColor">
           <property name="text">
            <string>Color</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Material color</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="_3">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>3</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QColorPushButton" name="TBEleAmbient">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Click to change color.</string>
              </property>
              <property name="whatsThis">
               <string>Click on this button will invoke color dialog, where you can change colors for all properties. Ambient, diffuse, emission, specular.</string>
              </property>
              <property name="text">
               <string>Ambient</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QColorPushButton" name="TBEleDiffuse">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Click to change color.</string>
              </property>
              <property name="whatsThis">
               <string>Click on this button will invoke color dialog, where you can change colors for all properties. Ambient, diffuse, emission, specular.</string>
              </property>
              <property name="text">
               <string>Diffuse</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QColorPushButton" name="TBEleEmission">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Click to change color.</string>
              </property>
              <property name="whatsThis">
               <string>Click on this button will invoke color dialog, where you can change colors for all properties. Ambient, diffuse, emission, specular.</string>
              </property>
              <property name="text">
               <string>Emission</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QColorPushButton" name="TBEleSpecular">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Click to change color.</string>
              </property>
              <property name="whatsThis">
               <string>Click on this button will invoke color dialog, where you can change colors for all properties. Ambient, diffuse, emission, specular.</string>
              </property>
              <property name="text">
               <string>Specular</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Shininess:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="SBEleShininess">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="maximum">
               <number>128</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QColorPushButton</class>
   <extends>QPushButton</extends>
   <header location="global">ColorPushButton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
